bpf/mock/generate_headers.sh

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021 Authors of Cilium
+
+# script to auto-generate helpers.h, helpers_skb.h, and helpers_xdp.h from bpf/helpers.h, bpf/helpers_skb.h, and bpf/helpers_xdp.h
+
+#!/bin/bash
+
```

Comment:
ditto, would be great if you could add some comment describing the desired operation